config/mtl.toml

```diff
@@ -393,3 +393,23 @@ count = 15
 
 	# mod_cfg [PAR_0 PAR_1 PAR_2 PAR_3 IS_BYTES CPS IBS OBS MOD_FLAGS CPC OBLS]
 	mod_cfg = [0, 0, 0, 0, 14400, 1114000, 16, 16, 0, 0, 0]
+
+	# smart amp test module config
+	[[module.entry]]
+	name = "SMATEST"
+	uuid = "167A961E-8AE4-11EA-89F1-000C29CE1635"
+	affinity_mask = "0x1"
+	instance_count = "1"
+	domain_types = "0"
+	load_type = "0"
+	module_type = "9"
```

Comment:
The module_type is 8 in [tgl-cavs.toml](https://github.com/thesofproject/rimage/blob/main/config/tgl-cavs.toml#L278). Is it correct?

Reply:
Lets align all TGL to use MTL module types.

Reply:
It should be 13 
````
enum mod_type {
        ebasefw = 0,
        emixin,
        emixout,
        ecopier,
        epeakvol,
        eupdwmix,
        emux,
        esrc,
        ewov,
        efx,
        eaec,
        ekpb,
        emicselect,
        efxf,   /*i.e.SmartAmp */
        eaudclass,
        efakecopier,
        eiodriver,
        ewhm,
        egdbstub,
        esensing,
        emax,
        einvalid = emax
} ;
````

Reply:
updated the module type for smart amp test to efxf (13).  the module types used between mtl and tgl are not aligned, ex, GAIN, for tgl, module_type=5, for mtl module_type=9, I will send another patch to make it aligned.